grpc-example/grpc-example-scala/project/plugins.sbt

```diff
@@ -1,5 +1,12 @@
+
+enablePlugins(BuildInfoPlugin)
+val playGrpcV = "0.8.1"
+buildInfoKeys := Seq[BuildInfoKey]("playGrpcVersion" -> playGrpcV)
```

Comment:
The gRPC version is using in the plugin file and in the project itself. To avoid adding the version in many places, I'm adding it on BuildInfo and reusing.